CC/Sounder/recorder_worker.cc

```diff
@@ -230,7 +246,7 @@ herr_t RecorderWorker::initHDF5()
         H5::DataSpace pilot_dataspace(kDsDim, dims_pilot, max_dims_pilot);
         this->file_->createDataSet("/Data/Pilot_Samples",
             H5::PredType::STD_I16BE, pilot_dataspace, this->pilot_prop_);
-        pilot_dataspace.close();
+        // pilot_dataspace.close(); // OBCH FIXME - seems to be done during garbage collection gc() func
```

Comment:
This looks to be ok to remove.  The H5::DataSpace destructor should call close.  As time allows, we should remove all the close() calls on  H5::DataSpace objects or add them to all of them.